indirect.h

```diff
@@ -343,4 +347,29 @@ struct std::hash<xyz::indirect<T, Alloc>> {
   }
 };
 
+#if (__cpp_lib_format >= 201907L)
+
+namespace xyz {
+
+template <class T>
+concept is_formattable = requires(T t) { std::formatter<T, char>; };
+
+}
+
+template <class T, class Alloc>
+  requires xyz::is_formattable<T>
+struct std::formatter<xyz::indirect<T, Alloc>> {
+  constexpr auto parse(format_parse_context& ctx)
+      -> format_parse_context::iterator {
+    return std::begin(ctx);
+  }
+
+  auto format(xyz::indirect<T, Alloc> const& value, format_context& ctx) const
+      -> format_context::iterator {
+    using namespace std::literals;
+    return std::format_to(ctx.out(), "{}", *value);
```

Comment:
Presumably propagation of `format_context` is required here... needs testing once implemented.

Reply:
This is working now